lib_nbgl/src/nbgl_use_case.c

```diff
@@ -1165,6 +1211,7 @@ void nbgl_useCaseStatus(const char *message, bool isSuccess, nbgl_callback_t qui
                                            .centeredInfo.text2   = NULL,
                                            .centeredInfo.text3   = NULL,
                                            .tapActionText        = "",
+                                           .isSwipe              = false,
```

Comment:
At first sight, `isSwipe` naming doesn't help me to understand the behavior behind?